pkg/network/tracer/tracer_linux_test.go

```diff
@@ -2796,3 +2803,75 @@ func skipEbpflessTodo(t *testing.T, cfg *config.Config) {
 		t.Skip("TODO: ebpf-less")
 	}
 }
+
+const (
+	srvAddr = "127.0.0.1:7777"
+)
+
+func TestMapCleanerDoesNotRemoveNewConnections(t *testing.T) {
+	// Reproduction for incident-34000
+	srv, cancel := grpc.NewGRPCTLSServer(t, srvAddr, true)
+	t.Cleanup(cancel)
+	defaultCtx := context.Background()
+
+	cfg := testConfig()
+	cfg.ServiceMonitoringEnabled = true
```

Comment:
I think the map cleaner still runs with NPM enabled and USM disabled (default config), so if you are just testing the connection protocol map cleaner you shouldn't need to set any of these. but happy to leave it due to the urgency of the incident

Reply:
That's true!
The test was written by me in the incident channel, and just resued other USM tests
I think that for the sake of 7.62, it does not matter
But for main branch, we should create a simpler test, with NPM only (and perhaps a similar version with USM)